.travis.yml

```diff
@@ -106,17 +106,18 @@ jobs:
       before_script:
         - gcloud docker --authorize-only
         - bin/docker-pull-deps
+        - bin/docker-pull master
         - |
-          export CONDUIT_TAG=$(. bin/_tag.sh ; clean_head_root_tag)
+          export CONDUIT_TAG=$(bin/root-tag)
```

Comment:
the one small difference here is that this could in theory produce a dirty tag.  perhaps root-tag should support a mode that cannot produce dirty tags?